### DISCUSSION TITLE
Add an option to be able to move roms to a subfolder.

### DISCUSSION THREAD

**Discussion by hbooo** [RandomNinjaAtk/docker-raromprocessor#13] [Ideas]
Hi,

I already talked about this in discord but as it's close now I would like to talk about this again. 

I know that you may develop an all in one docker with emulatorjs and that's great! In the meantime, would it be possible to have an option to be able to move roms from output in a specifix subfolder? I already do it using a script but got roms in double because your script expect roms to be in root console folder

**Comment by RandomNinjaAtk** [comment]
Can you explain more on why and what you need specific sub folder?

I'm not aware of any platforms that use them/prefer it that way.

Big Box/launch box
Retropi/batocera/recalbox 
Retroarch
Emulatorjs

All don't use sub folders.

Also give an example of what you think it should look like and the use case.

**Comment by hbooo** [comment]
I'm using emulatorjs. It expects to have roms in a "roms" folder (taken from homepage) : 

> Rom Identification basics
> The basic steps for scanning in roms is:

> Place the roms in the "roms" directory for your system in your mounted /data/ directory.

I restart from a new folder and here the result for 3do : 
![image](https://user-images.githubusercontent.com/242125/182111275-149ad21e-00ae-44e4-a38f-588cd60525d2.png)

Plumbers Don't Wear Ties.zip is in the root of 3do folder so emulatorjs doesn't detect the rom. I have to manually move the roms to roms/ folder in order to be able to scan and play with it in emulatorjs.
